Symfony/CS/Fixer/Symfony/NoEmptyLinesAfterPhpdocsFixer.php

```diff
@@ -26,13 +26,24 @@ class NoEmptyLinesAfterPhpdocsFixer extends AbstractFixer
      */
     public function fix(\SplFileInfo $file, $content)
     {
+        static $forbiddenSuccessors = array(
+            T_DOC_COMMENT,
+            T_COMMENT,
+            T_WHITESPACE,
```

Comment:
Whitespace was already here. And the reason is https://github.com/FriendsOfPHP/PHP-CS-Fixer/commit/aedf9ab5f73131f1ef75e20d37acea564afcc09e